tidb.go

```diff
@@ -243,9 +243,24 @@ func NewStore(uri string) (kv.Storage, error) {
 
 var queryStmtTable = []string{"explain", "select", "show", "execute", "describe", "desc"}
 
+func trimSQL(sql string) string {
```

Comment:
@ngaut We can handle the sql in #199 now. For sql text start with --, we just need to drop the entire sql text and do nothing else.

➜  interpreter git:(shenli/comment-in-query) ✗ cat xx
 /* mysql-connector-java-5.1.35 ( Revision: 5fb9c5849535c13917c2cf9baaece6ef9693ef27 ) */SHOW VARIABLES WHERE Variable_name ='language' OR Variable_name = 'net_write_timeout' OR Variable_name = 'interactive_timeout' OR Variable_name = 'wait_timeout' OR Variable_name = 'character_set_client' OR Variable_name = 'character_set_connection' OR Variable_name = 'character_set' OR Variable_name = 'character_set_server' OR Variable_name = 'tx_isolation' OR Variable_name = 'transaction_isolation' OR Variable_name = 'character_set_results' OR Variable_name = 'timezone' OR Variable_name = 'time_zone' OR Variable_name = 'system_time_zone' OR Variable_name = 'lower_case_table_names' OR Variable_name = 'max_allowed_packet' OR Variable_name = 'net_buffer_length' OR Variable_name = 'sql_mode' OR Variable_name = 'query_cache_type' OR Variable_name = 'query_cache_size' OR Variable_name = 'license' OR Variable_name = 'init_connect';
➜  interpreter git:(shenli/comment-in-query) ✗ cat xx | ./interpreter 
Welcome to the TiDB.
Version:
Git Commit Hash: None
UTC Build Time:  None

+--------------------------+------------------------+
| Variable_name            | Value                  |
+--------------------------+------------------------+
| character_set_results    | latin1                 |
| init_connect             |                        |
| character_set_client     | latin1                 |
| tx_isolation             | REPEATABLE-READ        |
| interactive_timeout      | 28800                  |
| max_allowed_packet       | 4194304                |
| lower_case_table_names   | 2                      |
| character_set_connection | latin1                 |
| time_zone                | SYSTEM                 |
| sql_mode                 | NO_ENGINE_SUBSTITUTION |
| query_cache_type         | OFF                    |
| net_write_timeout        | 60                     |
| system_time_zone         | CST                    |
| query_cache_size         | 1048576                |
| net_buffer_length        | 16384                  |
| license                  | GPL                    |
| wait_timeout             | 28800                  |
| character_set_server     | latin1                 |
+--------------------------+------------------------+

Bye